design/authenticators-status/authn_status_oidc.md

```diff
@@ -62,7 +60,12 @@ permissions on the webservice
     stronger than the pros. As we will still log the error, the operator will be able to 
     see the configuration failure and may proceed accordingly. This is relevant for all the 
     checks but this check's price is too high.
-
+- The group `conjur/authn-oidc/<service-id>/users` exists and has `read` & `authenticate` 
+permissions on the webservice
+    - Although it is a best practice to create a group with this name and add members who we want to have permissions to that group
```

Comment:
We want to be more detailed as to what 'group' we are talking about. Our goal is to explain that no, this additional group is not required but yes, this is a better option in terms of organizing your space.

For organizational purposes, we recommend that a new group is created for each authenticator.  That way, you can easily give the necessary `read` and `authenticate` privileges to multiple groups that already exist in Conjur. Although this is best practice, we do not enforce this in the product. Therefore, we cannot consider an authenticator 'unhealthy' if that extra grouping does not exist (or has insufficient permissions on the webservice) and will not include this as one of the status checks.